# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/videodev2.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual kind of user-space application fix.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.8.1/include/linux/videodev2.h.orig	2004-09-15 15:45:53.633962050 +0200
+++ linux-2.6.8.1/include/linux/videodev2.h	2004-09-15 15:45:47.348967172 +0200
@@ -17,6 +17,8 @@
 #include <linux/time.h> /* need struct timeval */
 #endif
 
+#include <linux/compiler.h>
+
 /*
  *	M I S C E L L A N E O U S
  */
